addons/website_quote/views/website_quotation.xml

```diff
@@ -148,7 +151,7 @@
                                   </a>
                               </div>
                           </div>
-                          <div class="text-center" t-if="need_payment" style="padding: 10px">
+                          <div class="text-center" t-if="need_payment or (quotation.state in ('sale') and quotation.require_payment and quotation.invoice_status != 'invoiced' and tx_state != 'done')" style="padding: 10px">
```

Comment:
@dpa-odoo Why do you check if quotation.invoice_status != 'invoiced' ?

I would display it whatever the invoice status. Here it means that you display the button if the quotation is in upselling, but not if it is invoiced ? It doesn't make sense to me..